src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -1319,6 +1319,7 @@ macro_rules! coverage_test_alias {
             }
 
             fn run(self, builder: &Builder<'_>) {
+                // Actually running the tests goes through a common helper method.
```

Comment:
This comment can be improved I think. It doesn't say enough about the context.

Reply:
Hmm, as think about how to explain this better, I’m realising that going through `Coverage` is unnecessarily complicated anyway.

If I change `run_unified_suite` to be a top-level helper function (with a better name), it should be easier to write a good comment. I’ll try it out tomorrow and see how it looks.